massa-execution-worker/src/context.rs

```diff
@@ -82,8 +82,8 @@ pub struct ExecutionContextSnapshot {
     /// address call stack, most recent is at the back
     pub stack: Vec<ExecutionStackElement>,
 
-    /// generated events during this execution, with multiple indexes
-    pub events: EventStore,
+    /// keep index of last event emitted in the context
+    pub last_event_index: usize,
```

Comment:
technically it's not the index of the last emitted event, but the index of the one just after that